src/Umbraco.Infrastructure/HostedServices/RecurringHostedServiceBase.cs

```diff
@@ -41,6 +43,13 @@ protected RecurringHostedServiceBase(ILogger logger, TimeSpan period, TimeSpan d
             _delay = delay;
         }
 
+        // Scheduled for removal in V11
+        [Obsolete("Please use constructor that takes an ILogger instead")]
+        protected RecurringHostedServiceBase(TimeSpan period, TimeSpan delay)
+        : this(StaticServiceProvider.Instance.GetRequiredService<ILogger>(), period, delay)
```

Comment:
```suggestion
        : this(StaticApplicationLogging.CreateLogger<RecurringHostedServiceBase>(), period, delay)
```

Reply:
was bad idea as the hosted services startup before the generic web host starts up (which inits static application logging).